src/java.desktop/windows/native/libawt/windows/awt_Canvas.cpp

```diff
@@ -214,7 +214,8 @@ void AwtCanvas::_SetEraseBackground(void *param)
     PDATA pData;
     JNI_CHECK_PEER_GOTO(canvas, ret);
 
-    AwtCanvas *c = (AwtCanvas*)pData;
+    AwtCanvas *c;
+    c = (AwtCanvas*) pData;
     c->m_eraseBackground = doErase;
     c->m_eraseBackgroundOnResize = doEraseOnResize;
```

Comment:
```suggestion
    {
        AwtCanvas *c = (AwtCanvas*) pData;
        c->m_eraseBackground = doErase;
        c->m_eraseBackgroundOnResize = doEraseOnResize;
    }
```
Does wrapping in {} work?  I think it looks better, and that's how we handle case labels in switch statements, isn't it?

Reply:
Wrapping it in a scope does work, but the problem is that it unfortunately becomes very messy and hard to review when the scope spans over a lot of lines (see below for instance, where I haven't switched from a scope to the split declaration and assignment yet). This approach was suggested by @djelinski in the review of https://github.com/openjdk/jdk/pull/15996